npm/qsharp/src/katas.ts

```diff
@@ -66,14 +66,17 @@ export type Kata = {
   id: string;
   title: string;
   sections: KataSection[];
+  published: boolean;
 };
 
-export async function getAllKatas(): Promise<Kata[]> {
-  return katasContent.katas as Kata[];
+export async function getAllKatas(includeUnpublished = false): Promise<Kata[]> {
```

Comment:
In case we add a bunch more optional switches to this method in the future, can we make this a named argument?

```suggestion
export async function getAllKatas({
  includeUnpublished = true,
}): Promise<Kata[]> {
```

And at the call site, it would say: `getAllKatas({ includeUnpublished: true })`


This is just setting ourselves up to avoid code like `getAllKatas(true, true, false, false, true)` down the line.

Reply:
Sounds good, I can do that (I'm assuming you're suggesting to change just the syntax, not the default value from false to true)